__tests__/app.test.js

```diff
@@ -0,0 +1,60 @@
+const request = require("supertest");
+const app = require("../app.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection.js");
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+afterAll(() => {
+    return db.end();
+});
+
+describe('ENDPOINT TESTS', () => {
+    describe('GET /api/topics', () => {
+        test('GET 200 - should return all an array with all topics from db', () => {
+            return request(app)
+                .get('/api/topics')
+                .expect(200)
+                .then((res) => {
+                    expect(res.body.topics).toBeInstanceOf(Array)
+                    expect(res.body.topics.length).toBe(3)
+                    res.body.topics.forEach((topic) => {
+                        expect(topic).toEqual(
+                            expect.objectContaining({
+                                slug: expect.any(String),
+                                description: expect.any(String)
+                            })
+                        );
+                    });
+                })
+        });
+        test('GET 200 - all topics should have correct structure', () => {
```

Comment:
Is this test doing anything additional to the previous test?

Reply:
Nope, not at the moment, the tests were supposed to have separate criteria but I accidentally duplicated part of the test. 

Corrected in #2f4ee939978a16495093c0c3db8f2770836dc208

Reply:
I don't know best to reference the changes sorry